src/m_menu.c

```diff
@@ -6441,6 +6473,16 @@ void M_StartControlPanel (void)
 
 void M_Drawer (void)
 {
+   if (setup_active && menu_background == 2)
+   {
+      int y;
+      byte *dest = screens[0];
```

Comment:
When I still had this in Crispy, it had a nice fade out effect. Just an idea. 

https://github.com/fabiangreffrath/crispy-doom/commit/cd807c0b5f76f5b09f119ff1bd8e463824a61bca

Reply:
I also like the shadowed text effect, why did you remove it? I think we can add it to Woof and bring it back to Crispy.

Reply:
> I also like the shadowed text effect, why did you remove it? I think we can add it to Woof and bring it back to Crispy.

It was deemed a pointless deviation from the Vanila experience. At this point, I decided that all UI that was already present in Vanilla should be preserved as far as possible, and only new ("added by Crispy") UI got some blink.